include/nbl/builtin/hlsl/bxdf/ndf/beckmann.hlsl

```diff
@@ -339,6 +339,8 @@ struct Beckmann
         if (isInfinity)
         {
             quant_type dmq;
+            dmq.microfacetMeasure = scalar_type(0.0);
+            dmq.projectedLightMeasure = scalar_type(0.0);
```

Comment:
both should be set to `bit_cast<scale_type>(numeric_limits<scale_type>::inf)` not 0s